plone/app/upgrade/v51/configure.zcml

```diff
@@ -300,6 +300,12 @@ Add image scaling options to image handling control panel.
             handler=".final.to517"
             />
 
+        <gs:upgradeDepends
+            title="Run to51beta4 upgrade profile."
+            description="Fixup image caption TinyMCE.
+                         Update plone-logged-in bundle"
+            import_profile="plone.app.upgrade.v51:to517"
+            />
```

Comment:
where does this come from?

Reply:
This can actually be removed because of: https://github.com/plone/plone.app.upgrade/blob/f42a09f47173aff76b338ba18ae9fff0dda3b207/plone/app/upgrade/v51/final.py#L68